src/main/kotlin/org/sert2521/infiniterecharge2020/climber/Climber.kt

```diff
@@ -20,11 +20,11 @@ class Climber : Subsystem("Climber") {
 
     // Ah. Now would be time to look away from the code (or at least not closely)
     // TODO: FIX THESE INCREDIBLE NAMES CORUTSEY OF MOISEUR WILL.I.AM
-    val topLimitSwitch = DigitalInput(BOTTOM_LIMIT_SWITCH)
-    val bottomLimitSwitch = DigitalInput(TOP_LIMIT_SWITCH)
+    val topLimitSwitch = DigitalInput(TOP_LIMIT_SWITCH)
+    val bottomLimitSwitch = DigitalInput(BOTTOM_LIMIT_SWITCH)
 
-    val atBottom get() = topLimitSwitch.get()
-    val atTop get() = !bottomLimitSwitch.get()
```

Comment:
Was it right to switch this and also get rid of `!` sign?

Reply:
Essentially, `atBottom` and `atTop` are returning the correct state of the climber currently. The `!` signs reverse the reading of the limit switch (which are needed depending on how the sensor is wired I believe?).  Again, the way the code is currently set up **works correctly**.

All we need to do is clean up the variable names so that they match. The tricky part is that there are two very confusing naming errors atm. It's a bit of a logic puzzle.